README.md

```diff
@@ -56,8 +76,92 @@ make lint
 make gen-docs
 ```
 
+### Debugging
+
+To debug the plugin in Intellij so that code can be stepped through with breakpoints, you will need:
```

Comment:
Is it possible to provide more generic debugging instructions that don't rely on a specific IDE?

Reply:
That's a good point - my experience was limited to Intellij.  I think it's difficult to come up with something general but I could say how this varies for other IDEs, eg. VSC.  Are there any other IDEs you use or you think are commonly in use apart from Intellij and VSC?